src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -329,14 +329,22 @@ function ReportActionItemMessageEdit(
         deleteDraft();
     }, [action, debouncedSaveDraft, deleteDraft, draft, reportID]);
 
+    const shouldUseEmojiPickerSelectionRef = shouldUseEmojiPickerSelection();
```

Comment:
Kind of weird to call this a ref, and I don't think we need to create a variable for this, let's just call it directly.